Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            string firstName = InputString("Enter First Name: ");
```

Comment:
No need a blank line between every statement. Use a blank line to separate unrelated blocks of code. For example: Reading is one thing, calculating is another, printing final result is third. (so separate each by 3 blanks lines in total).

Reply:
Good point - thanks for this kind of feedback.  I spent most of my programming writing code for myself so these kind of points help me to make my code more readable for someone who might take it on later.